### ISSUE TITLE
TreeSelect: Trigger Button missing aria-label

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#14355]
### Describe the bug

WCAG 2.1: Button missing a text alternative. Provide a text alternative for the button (.p-treeselect-trigger).

### Environment

Windows, Chrome

### Reproducer

https://primeng.org/treeselect

### Angular version

17.0.0

### PrimeNG version

17.0.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16.0
